### ISSUE TITLE
Bitnami Wordpress (AWS Lightsail) - Site becomes unresponsive / inaccessible 

### ISSUE THREAD

**Issue by PNAT** [bitnami/vms#466]
### Platform

AWS

### bndiagnostic ID [know more about bndiagnostic ID](https://docs.bitnami.com/general/how-to/understand-bndiagnostic/)

d81ca6e0-a4cf-252c-6f50-8189d43b8da6

### bndiagnostic output

? Apache: Found possible issues    
✓ Mariadb: No issues found    
? Connectivity: Found possible issues    
✓ Wordpress: No issues found    
? Resources: Found possible issues    
✓ Processes: No issues found    
✓ Php: No issues found

Apache Errors: 

Found recent error or warning messages in the Apache error log.
```
[Tue Nov 01 20:21:34.152158 2022] [proxy_fcgi:error] [pid 21234:tid 140687743670016] (70008)Partial results are valid but processing is incomplete: [client **ip_address**:57908] AH01075: Error dispatching request to : (reading input brigade) [Wed Nov 02 03:43:57.519638 2022] [autoindex:error] [pid 21234:tid Press [Enter] to continue:140687861167872] [client **ip_address**:51281] AH01276: Cannot serve directory /opt/bitnami/wordpress/wp-admin/css/: No matching DirectoryIndex (index.html,index.html,index.htm,index.php) found, and server-generated directory index forbidden by Options directive, referer: binance.com [Wed Nov 02 20:24:44.395136 2022] [autoindex:error] [pid 21234:tid 140688020629248] [client **ip_address**:50930] AH01276: Cannot serve directory /opt/bitnami/wordpress/wp-admin/css/: No matching DirectoryIndex (index.html,index.html,index.htm,index.php) found, and server-generated directory index forbidden by Options directive, referer: binance.com
```

### bndiagnostic was not useful. Could you please tell us why?

I can't figure out why the site is going offline? 

### Describe your issue as much as you can

Site = verbed.ca

After some period of time the site goes offline.  You can verify by going to verbed.ca. 

This has happened a handful of times.  I've been able to bring the site back by rebooting the AWS Lightsail instance.  But, I would like to find out why this is happening so I can (hopefully) correct the issue causing the outages. 

Many thanks for your help + support.

**Comment by PNAT** [comment]
Any chance it could be related to PageSpeed?  I just came across this video, but I haven't implemented it yet... 

https://www.youtube.com/watch?v=GoJ5taeZ0Ls&t=263s

**Comment by PNAT** [comment]
Thanks!  I rebooted the instance and all is well.

**Comment by PNAT** [comment]
Hey @gongomgra / Bitnami Support - 

A similar thing happened again, unfortunately.  I tried to run the Bitnami Diagnostic Tool, but it just polled and polled and never did anything. 

I ended up rebooting the Lightsail instance.  Is there any place I could manually download the logs and share them with you to see what happened?  OR can I run the diagnostic tool and it will show what happened retroactively?
